code-challenges.test.js

```diff
@@ -67,3 +161,24 @@ const vowelTester3 = "challenges"
 
 
 // b) Create the function that makes the test pass.
+
+const firstVowel = (vowel) => {
+    for(let i=0; i<vowel.length; i++) {
+        if(vowel[i] === "a" || 
+           vowel[i] === "e" || 
+           vowel[i] === "i" || 
+           vowel[i] === "o" || 
+           vowel[i] === "u") {
+            return i
+        }
+    }
+}
```

Comment:
Excellent work!  If you want to stretch yourself, think of some edge cases as well such as capitalized vowels.